content/blog/go-support-pulumi-2-0/index.md

```diff
@@ -195,30 +194,30 @@ func NewServiceDeployment(...){...}
 This allows us to simplify boilerplate and design strongly-typed APIs for our Kubernetes infrastructure (in this case, combining a `Service` and a `Deployment` into a single, simple, API with nice defaults for our use case):
 
 ```go
-                       _, err := NewServiceDeployment(ctx, "redis-leader", &ServiceDeploymentArgs{
-			Image: pulumi.String("k8s.gcr.io/redis:e2e"),
-			Ports: pulumi.IntArray{pulumi.Int(6379)},
-		})
+_, err := NewServiceDeployment(ctx, "redis-leader", &ServiceDeploymentArgs{
+Image: pulumi.String("k8s.gcr.io/redis:e2e"),
+Ports: pulumi.IntArray{pulumi.Int(6379)},
```

Comment:
The two lines with `Image` and `Port` should still be indented. 

![Screen Shot 2020-05-19 at 8 27 21 AM](https://user-images.githubusercontent.com/2553171/82345844-97e3cc00-99aa-11ea-906a-362adfc21ea4.png)

Reply:
@spara Looks like we did not address this comment?